VideoWeb/VideoWeb/ClientApp/src/app/shared/models/hearing-summary.spec.ts

```diff
@@ -19,6 +19,9 @@ describe('HearingSummary', () => {
         expect(hearing.scheduledStartTime).toEqual(c.scheduled_date_time);
         expect(hearing.scheduledEndTime).toBeDefined();
         expect(hearing.hearingVenueName).toBe(c.hearing_venue_name);
+        expect(hearing.startedDateTime).toEqual(c.started_date_time);
```

Comment:
it might be worth adding a test to verify the model returns a null if the start and end time is not set rather than risk errors being thrown